Automattic/jetpack
# Boost: Refactor performance-history into react

## Proposed changes:
* Fixed a width issue on the history graph where the graph wouldn't get smaller if the container shrunk
* Refactor performance history into react and get rid of History.svelte component
* Implemented ErrorNotice component in react which is a dependency of performance history. Fixed broken style while I'm at it.

### Other information:


## Jetpack product discussion
pc9hqz-2jy-p2

## Does this pull request change what data or activity we track or use?
No

## Testing instructions:
Check the performance history panel using boost premium and make sure it's working
